src/main/java/org/spongepowered/api/world/volume/game/SignalAwareVolume.java

```diff
@@ -0,0 +1,322 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.world.volume.game;
+
+import org.spongepowered.api.util.Direction;
+import org.spongepowered.api.world.SignalType;
+import org.spongepowered.math.vector.Vector3i;
+
+import java.util.Objects;
+import java.util.function.Supplier;
+
+/**
+ * A volume aware of the signal that can "power" blocks. <br>
+ * This signal is often referred to as "redstone signal".
+ */
+public interface SignalAwareVolume extends PrimitiveGameVolume {
+
+    /**
+     * Returns whether the given position can conduct the signal.
```

Comment:
Is there any implicit direction being mentioned here? Does direction actually matter in this case?